test/functional/mempool_packages.py

```diff
@@ -31,10 +31,8 @@ def set_test_params(self):
         self.noban_tx_relay = True
         self.extra_args = [
             [
-                "-maxorphantx=1000",
             ],
```

Comment:
nit:
```python
        self.extra_args = [
            [],
            ...
```

leaving

```python
        self.extra_args = [
            [
            ],
            ...
```

is a bit weird

Reply:
I don't see this as a problem per se. [test/functional/feature_rbf.py](https://github.com/bitcoin/bitcoin/blob/058af75874ffa2b4064e3d6d30cc50f0ec754ba8/test/functional/feature_rbf.py#L38) already had the same pattern.

Reply:
Fair, it's a stylistic nit so just an opinion, can be disregarded